src/FSharpSource.targets

```diff
@@ -230,6 +233,7 @@
     <DefineConstants>$(DefineConstants);NO_LOADER_OPTIMIZATION</DefineConstants>
     <DefineConstants>$(DefineConstants);SILVERLIGHT_COMPILER_FSHARP_CORE</DefineConstants>
     <DefineConstants>$(DefineConstants);I_DONT_KNOW_HOW_TO_DO_THIS_YET</DefineConstants>
+    <DefineConstants>$(DefineConstants);NETSTANDARD1_5</DefineConstants>
```

Comment:
Does the generated code need to be different?

From: Enrico Sada [mailto:notifications@github.com]
Sent: Wednesday, April 13, 2016 4:32 PM
To: Microsoft/visualfsharp <visualfsharp@noreply.github.com>
Cc: Kevin Ransom <Kevin.Ransom@microsoft.com>
Subject: Re: [Microsoft/visualfsharp] Revert the revert of fssrgen (#1064)


In src/FSharpSource.targets<https://github.com/Microsoft/visualfsharp/pull/1064#discussion_r59643851>:

> @@ -230,6 +233,7 @@

>      <DefineConstants>$(DefineConstants);NO_LOADER_OPTIMIZATION</DefineConstants>

>      <DefineConstants>$(DefineConstants);SILVERLIGHT_COMPILER_FSHARP_CORE</DefineConstants>

>      <DefineConstants>$(DefineConstants);I_DONT_KNOW_HOW_TO_DO_THIS_YET</DefineConstants>

> +    <DefineConstants>$(DefineConstants);NETSTANDARD1_5</DefineConstants>

the generated file from fssrgen contains an #if NETSTANDARD1_5 so the same file can work for both net46 and netstandard1.5 (dnxcore50 too, but now is useless)
NETSTANDARD1_5 it's the default define added for netstandard1.5 framework (like NET46 for .net 4.6)

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/visualfsharp/pull/1064/files/9e72d0c1b22f2a8d91e7e7e047def11637acb773#r59643851>